app-backend/tool/src/main/scala/ast/MapAlgebraAST.scala

```diff
@@ -84,6 +82,87 @@ object MapAlgebraAST {
     def withArgs(newArgs: List[MapAlgebraAST]): MapAlgebraAST = copy(args = newArgs)
   }
 
+  case class Max(args: List[MapAlgebraAST], id: UUID, metadata: Option[NodeMetadata]) extends Operation {
```

Comment:
They were just moved around within the file to preserve organizational structure between local/focal ops